tests/compile/ctuple_6454.pyx

```diff
@@ -0,0 +1,16 @@
+# mode: compile
```

Comment:
I'd prefer a runnable test that actually verifies the correct assignment mapping.